.github/workflows/github_actions.yml

```diff
@@ -0,0 +1,37 @@
+name: Coverity scan
+
+on: [push, pull_request] 
+
+jobs:
+  coverity-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: install dependencies
+        run: |
+         bash .ci/setup.sh 
+      - name: Download Coverity Build Tool
+        run: |
+          wget -q https://scan.coverity.com/download/cxx/linux64 --post-data "token=$TOKEN&project=DLTcollab/tangle-accelerator" -O cov-analysis-linux64.tar.gz
```

Comment:
Move the `coverity` related commands to a dedicated shell script.

Reply:
Once you prepare a dedicated coverity script, you can trigger the associated operations via the bash functions predefined within the script.